magento/pwa-studio
# [docs] Tutorial to create storefront component

## Description

This is a step-by-step tutorial for creating a TagList component for a PWA Studio storefront.

## Related Issue
Closes PWA-124

## Acceptance 
@jimbo 

### Verification Stakeholders
@jimbo 

### Specification



### Verification Steps
1. Navigate to the `pwa-devdocs` directory
2. Lint the new file: `npm run lint src/tutorials/create-tag-list-component/index.md`
3. Verify no linting errors
4. Build HTML preview: `npm run deploy`
5. Navigate to preview site
6. Navigate to the Tutorials section
7. Verify new tutorial topic appears in table of content
8. Navigate to the new topic
9. Follow instructions for creating a new component
10. Verify steps result in a working TagList component as described in the tutorial

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
